### PR TITLE
Stand-alone Toolbar class

### PR DESCRIPTION
**Pull request by afshin**
The best way forward for this was to start over after the big CSS merge.

Fixes https://github.com/jupyter/jupyterlab/issues/844

### CODE REVIEW

**Comment by ellisonbg** [comment]
cool i will have a look

**Comment by afshin** [comment]
@ellisonbg FYI, this is still work-in-progress.

File: src/notebook/notebook/widgetfactory.ts [hunk 1]
```diff
@@ -87,7 +87,6 @@ class NotebookWidgetFactory extends ABCWidgetFactory<NotebookPanel, INotebookMod
     panel.context = context;
     ToolbarItems.populateDefaults(panel);
     this.widgetCreated.emit(panel);
-    console.log(panel.content.childAt(0).editor);
```
Comment (blink1073) [inline src/notebook/notebook/widgetfactory.ts]: Ha, I was wondering where this log was coming from.

File: src/toolbar/index.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  text-align: center;
+  display: inline-block;
+}
+
+
+.jp-Toolbar-item.jp-Toolbar-cellType select {
+    background: #FFFFFF;
+}
+
+.jp-Toolbar-cellType .jp-Toolbar-cellTypeDropdown {
```
Comment (blink1073) [inline src/toolbar/index.css]: missing a line

Comment (blink1073) [inline src/toolbar/index.css]: some of these are specific to notebook as well.

**Comment by blink1073** [comment]
LGTM, thanks!

**Pull request merged**